utilities/plugins/openai_plugin.py

```
@@ -95,7 +96,8 @@ def streaming_request_http(
         tokens = []
         try:
             url = f"{self.host}{endpoint}"
-            response = requests.post(url, headers=headers, json=data, verify=False, stream=True)
+            LOGGER.info(f"Streaming request to {url} with timeout=1200s")
+            response = requests.post(url, headers=headers, json=data, verify=False, stream=True, timeout=(60, 1200))
```

Ruff (0.15.12)

[error] Probable use of `requests` call with `verify=False` disabling SSL certificate checks

(S501)